README.md

```diff
@@ -1,5 +1,24 @@
 # Research User Group
 
-Twice-monthly meetings on [second Wednesdays at 09:00 UTC](https://calendar.google.com/event?action=TEMPLATE&tmeid=NGhsdnFxZG1jbDZsaTllamVnZjdkZmduNG9fMjAxOTA4MTRUMDkwMDAwWiBjaHVuZ0BsaW51eGZvdW5kYXRpb24ub3Jn&tmsrc=chung%40linuxfoundation.org&scp=ALL), and [fourth Wednesdays at 15:00 UTC](https://calendar.google.com/event?action=TEMPLATE&tmeid=N3U5NjhubHJpaTRtZHAyNWlqY2FzZjljc2NfMjAxOTA4MjhUMTUwMDAwWiBjaHVuZ0BsaW51eGZvdW5kYXRpb24ub3Jn&tmsrc=chung%40linuxfoundation.org&scp=ALL).
+The CNCF Research User Group’s purpose is to function as a focal point for the
+discussion and advancement of Research Computing using “Cloud Native”
+technologies. This includes enumerating current practices, identifying gaps,
+and directing effort to improve the Research Cloud Computing ecosystem.
 
-Agenda: https://docs.google.com/document/d/1vvXxW4Cd4P5gcmWGz-_yKbgJex2_NlSWaHtsk_56TnA/edit?ts=5d53c5ff#
+Twice-monthly meetings on the [second Wednesdays at 09:00 UTC][1], and the
+[fourth Wednesdays at 15:00 UTC][2].
+
+- [Meeting Agenda and Notes][Agenda]
```

Comment:
how about add links, e.g.  [Meeting Agenda and Notes]( https://docs.google.com/document/d/1vvXxW4Cd4P5gcmWGz-_yKbgJex2_NlSWaHtsk_56TnA/edit?ts=5d53c5ff# )

Reply:
Fixed! Thanks for catching that. I thought I had done that with a reference link via `[Agenda]` 🤦‍♂

Reply:
lgtm, thanks :)